go/cmd/vtctldclient/internal/command/shards.go

```diff
@@ -52,6 +53,41 @@ var (
 		Args: cobra.ExactArgs(2),
 		RunE: commandRemoveShardCell,
 	}
+	// SetShardIsPrimaryServing makes a SetShardIsPrimaryServing gRPC call to a
+	// vtctld.
+	SetShardIsPrimaryServing = &cobra.Command{
+		Use:                   "SetShardIsPrimaryServing <keyspace/shard> <true/false>",
+		Short:                 "Add or remove a shard from serving. This is meant as an emergency function. It does not rebuild any serving graphs; i.e. it does not run `RebuildKeyspaceGraph`.",
+		DisableFlagsInUseLine: true,
+		Args:                  cobra.ExactArgs(2),
+		RunE:                  commandSetShardIsPrimaryServing,
+	}
+	// SetShardTabletControl makes a SetShardTabletControl gRPC call to a vtctld.
+	SetShardTabletControl = &cobra.Command{
+		Use:   "SetShardTabletControl [--cells=c1,c2...] [--denied-tables=t1,t2,...] [--remove] [--disable-query-service[=0|false]] <keyspace/shard> <tablet_type>",
+		Short: "Sets the TabletControl record for a shard and tablet type. Only use this for an emergency fix or after a finished vertical split. The MigrateServedFrom and MigrateServedType commands set this record appropriately already.",
```

Comment:
Should we be using MoveTables / Reshard instead of vertical and horizontal split?